src/adapter/activateDaffodilDebug.ts

```diff
@@ -291,21 +286,29 @@ class DaffodilConfigurationProvider
   ): ProviderResult<DebugConfiguration> {
     // if launch.json is missing or empty
     if (!config.type && !config.request && !config.name) {
-      const editor = vscode.window.activeTextEditor
-      if (editor && editor.document.languageId === 'xml') {
-        config.type = 'dfdl'
-        config.name = 'Launch'
-        config.request = 'launch'
-        config.program = '${file}'
-        config.data = '${command:AskForDataName}'
-        config.stopOnEntry = true
-        config.useExistingServer = false
-        config.infosetOutput = {
-          type: 'file',
-          path: '${file}-infoset.xml',
-        }
-        config.debugServer = 4711
-      }
+      config = getConfig('Launch', 'launch', 'dfdl')
+      // config.type = 'dfdl'
```

Comment:
can this commented out code be removed?

Reply:
yes I completely forgot about it will remove it now